doc/devel/documenting_mpl.rst

```diff
@@ -10,75 +10,73 @@ Getting started
 Installing dependencies
 -----------------------
 
-The documentation for matplotlib is generated from reStructuredText using the
-Sphinx_ documentation generation tool. There are several extra requirements that
-are needed to build the documentation. They are listed in the file
-`doc-requirements.txt <https://github.com/matplotlib/matplotlib/blob/master/doc-requirements.txt>`_
-as well as listed below:
+The documentation for Matplotlib is generated from reStructuredText using
+the Sphinx_ documentation generation tool.  There are several extra
+requirements that are needed to build the documentation.  They are listed in
+:file:`doc-requirements.txt` as well as listed below:
 
-1. Sphinx-1.3 or later(Version 1.5.0 is not supported)
+1. Sphinx 1.3 or later (1.5.0 is not supported)
 2. numpydoc 0.4 or later
 3. IPython
-4. Mock
+4. mock
 5. colorspacious
-6. pillow
-7. graphviz
+6. Pillow
+7. Graphviz
 
 .. note::
 
   * You'll need a minimal working LaTeX distribution for many examples to run.
-  * `Graphviz <http://www.graphviz.org/Download.php>`_ is not a python package, and needs
-    to be installed separately.
+  * `Graphviz <http://www.graphviz.org/Download.php>`_ is not a Python package,
+    and needs to be installed separately.
 
 General structure
 -----------------
 
-All documentation is built from the :file:`doc/` directory. This directory contains both
-``.rst`` files that contain pages in the documentation, directories that contain more
-``.rst`` files, and configuration files for Sphinx_.
+All documentation is built from the :file:`doc/` directory.  This directory
+contains both ``.rst`` files that contain pages in the documentation,
+directories that contain more ``.rst`` files, and configuration files for
+Sphinx_.
 
 .. note::
 
-  An exception to this are the directories :file:`gallery` and :file:`tutorials`, which
-  exist in the root directory. These contain Python files that are built by `Sphinx Gallery`_.
-  When the docs are built, directories of the same name will be generated inside of :file:`docs/`.
-  The generated directories :file:`docs/gallery` and :file:`docs/tutorials` can be safely deleted.
+   An exception to this are the directories :file:`gallery` and
```

Comment:
Should be `examples` instead of `gallery` I think now?

Reply:
No, the files end up in the gallery subfolder of the source tree.